### PR TITLE
To Do List: Part 2 interactive list

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I: 
- Added professional Readme file
- Fixed linters errors
- Set up the project with webpack
- Created HTML dynamically with HTML webpack Plugins
- Modified structure of webpack for solving issues
- Added basic structure & linters checks
- Added another script file
- Added other functions to the project

### CODE REVIEW

File: src/status.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    } else {
+      this.taskList = JSON.parse(localStorage.getItem('taskList'));
+    }
+  }
+
+    saveStorage = () => {
+      localStorage.setItem('taskList', JSON.stringify(this.taskList));
+    }
+
+    mark = (index) => {
```
Comment (Mihndim2020) [inline src/status.js:30]: - [x] Please add a event listener (`change`) to the checkbox to track the changes. The task object's values for `completed` is updated upon user actions.

File: src/status.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      const task = this.taskList[index];
+      task.completed = true;
+      this.saveStorage();
+    }
+
+    unmark = (index) => {
```
Comment (Mihndim2020) [inline src/status.js:36]: - [x] Please add a event listener (`change`) to the checkbox to track the changes. The task object's values for `completed` is updated upon user actions.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 This is a project that I create to practice webpack. 
 
-![screenshot](https://user-images.githubusercontent.com/73128809/144694338-9c742cfa-d277-4e3b-90c4-10030b9852a7.png)
+![screenshot](https://user-images.githubusercontent.com/73128809/144714616-ef209345-3532-4e86-9397-03a51d2b8f1f.png)
 
 The purpose of this project is create a to do list including webpack.
```
Comment (Mihndim2020) [inline README.md:9]: - [x] Please you app is not running from my end, make sure the app is running, and kindly add steps to clone and run the app locally. See image below

devserve
.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Great job with UI

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/status.js (hunk 2 above)
Comment (ivaan1992) [reply src/status.js:36]: Captura de pantalla de 2021-12-04 12-14-41

I added the event listener change in the index script file.

**Review by D4nielJ — APPROVED** [review]
## Hello @ivaan1992, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**